samples/grpc/local-drone-control-scala/src/main/resources/prod.conf

```diff
@@ -0,0 +1,39 @@
+# Production configuration for running the local-drone-control service in Kubernetes,
```

Comment:
Maybe we should not name this `prod.conf` since the single node application.conf is also a prod alternative. Maybe `application-kubernetes.conf`?

Reply:
Sounds good. Was going to name it `kubernetes.conf` originally.

Reply:
Although, I guess the single node with H2 version can run in kubernetes too.

Reply:
true, but then we should have rather different deployment yml for that so that it's not forming a cluster with other nodes